<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="VerifyProjNameLength"
           tests="3"
           failures="2"
           errors="0"
           skipped="1"
           time="0.000">
  <properties />
            <testcase name="checkTestSkip" time="0.000" classname="com.uat.suite.tm_project.VerifyProjNameLength">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.uat.util.Xls_Reader.getRowCount(Xls_Reader.java:46)
  com.uat.util.TestUtil.isTestCaseRunnable(TestUtil.java:39)
  com.uat.suite.tm_project.VerifyProjNameLength.checkTestSkip(VerifyProjNameLength.java:50)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
  org.testng.TestRunner.beforeRun(TestRunner.java:641)
  org.testng.TestRunner.run(TestRunner.java:609)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
      </failure>

          </testcase>
            <testcase name="verifyProjNameLength" time="0.000" classname="com.uat.suite.tm_project.VerifyProjNameLength">
                <failure type="java.lang.RuntimeException"
                  message="java.lang.NullPointerException"
              >
<![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
  org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
  org.testng.internal.Parameters.handleParameters(Parameters.java:429)
  org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
  org.testng.internal.Invoker.createParameters(Invoker.java:1075)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.NullPointerException
  com.uat.util.Xls_Reader.isSheetExist(Xls_Reader.java:422)
  com.uat.util.TestUtil.getData(TestUtil.java:62)
  com.uat.suite.tm_project.VerifyProjNameLength.getTestData(VerifyProjNameLength.java:278)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
  org.testng.internal.Parameters.handleParameters(Parameters.java:429)
  org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
  org.testng.internal.Invoker.createParameters(Invoker.java:1075)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
      </failure>

          </testcase>
  
            <testcase name="reportTestResult" time="0.0" classname="com.uat.suite.tm_project.VerifyProjNameLength">
          <skipped />
    </testcase>
  
  
</testsuite>
